global-script.lua

```diff
@@ -3312,3 +3467,297 @@ function tCompare(t1,t2)
     end
     return cc2(t1) == cc2(t2)
 end
+
+function swapPlayerAreaColors(a, b)
```

Comment:
looks like `tintSwap` and `updateBlock` local functions aren't being used anymore?

Reply:
Good catch.